lib/krb5/fcache.c

```diff
@@ -1240,7 +1240,7 @@ fcc_lastchange(krb5_context context, krb5_ccache id, krb5_timestamp *mtime)
 {
     krb5_error_code ret;
     struct stat sb;
-    int fd = 0;
+    int fd = -1;
```

Comment:
'fd' is assigned -1 upon entry to line fcc_open() at line 1245.   It shouldn't be initialized.